spring-cloud-kubernetes-fabric8-config/src/main/java/org/springframework/cloud/kubernetes/fabric8/config/Fabric8ConfigMapPropertySource.java

```diff
@@ -57,8 +57,7 @@ private static SourceData getSourceData(Fabric8ConfigContext context) {
 	// we need to pass various functions because the code we are interested in
 	// is protected in ConfigMapPropertySource, and must stay that way.
 	private static Fabric8ContextToSourceData namedConfigMap() {
```

Comment:
these two functions: `ConfigMapPropertySource::getSourceName, ConfigMapPropertySource::withPrefix` have been removed from here and instead it delegates to `ConfigUtils`. This was needed because secrets implementation use these methods also